livox_ros_driver/livox_ros_driver/lddc.cpp

```diff
@@ -251,7 +237,111 @@ uint32_t Lddc::PublishPointcloud2(LidarDataQueue *queue, uint32_t packet_num,
   return published_packet;
 }
 
-void Lddc::FillPointsToPclMsg(PointCloud::Ptr& pcl_msg, \
+void Lddc::FillPointsToPointCloudXYZIRT(PointCloudXYZIRT::Ptr& pcl_msg, \

```

Comment:
Do we need these back slashes? I understand the original code has them, but ...

Reply:
I removed them.